myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,243 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+  before do
+    create(:user)
+  end
+
+  describe '#index' do
+    before do
+      visit root_path
+    end
+
+    it 'display default item' do
+      expect(page).to have_link 'Add Task'
+    end
+
+    it 'go to New Task page' do
+      click_on 'Add Task'
+      expect(current_path).to eq new_task_path
+    end
+
+    context 'when user has tasks' do
+      before do
+        create(:task)
+        visit current_path
+      end
+
+      it "return user's task" do # rubocop:disable RSpec/MultipleExpectations
+        expect(page).to have_link '散歩'
+        expect(page).to have_link 'Edit'
+        expect(page).to have_link 'Delete'
+      end
+    end
+
+  end
+
+  describe '#show' do
+    let(:task) { create(:task) }
+
+    before do
+      visit task_path(task)
+    end
+
+    it 'return task info' do # rubocop:disable RSpec/MultipleExpectations
+      expect(page).to have_content task.name
+      expect(page).to have_content task.description
+      expect(page).to have_content task.priority
+      expect(page).to have_content task.limit
+    end
+
+    context 'when user click Edit' do
+      it 'return edit task form' do
+        click_on 'Edit'
+        expect(page).to have_current_path edit_task_path(task), ignore_query: true
+      end
+    end
+
+    context 'when user click Delete' do
+      it 'delete task' do
+        expect {
+          click_on 'Delete'
+        }.to change { Task.count }.by(-1)
+      end
+
+      it 'display delete success message' do
+        click_on 'Delete'
+        expect(page).to have_content 'delete success'
+      end
+    end
+
+    context 'when user click Home' do
+      it 'go to roog page' do
+        click_on 'Home'
+        expect(current_path).to eq root_path
```

Comment:
こんな感じでも書けそうですー（rubocop標準だと下記のような書き方が推奨だったりします）

```suggestion
        expect(page).to have_current_path root_path, ignore_query: true
```

Reply:
上と同じく修正しました！
https://github.com/Fablic/training/pull/1330/commits/80cdd67e72f22d7e13cdb4e9f2bf8d668deea26a